tests/tck/features/optimizer/IndexScanRule.feature

```diff
@@ -93,9 +93,10 @@ Feature: Match index selection
     And the execution plan should be:
       | id | name           | dependencies | operator info |
       | 6  | Project        | 2            |               |
-      | 2  | AppendVertices | 5            |               |
-      | 5  | IndexScan      | 0            |               |
-      | 0  | Start          |              |               |
+      | 9  | Filter         | 3            |               |
```

Comment:
Why add Filter node?

Reply:
it's a bug, otherwise the filter information will be lost after index degradation

Reply:
Aren't all filter items embedded in IndexScan node? Why will the filter info be lost?

Reply:
What if the index information does not match? filter info should be preserved even if embedded

Reply:
> when embedding filter into the indexScan, it does not check whether the index match The purpose of retaining the filter is to prevent the loss of filter information after index degradation

ACK. Related issue: https://github.com/vesoft-inc/nebula/issues/3715